docs/examples/no_block.md

```diff
@@ -0,0 +1,32 @@
+# Non-blocking architecture
+
+
+Create a pywebview window in a separate process, so that main thread is not blocked.
+
+
+``` python
+import webview
+import multiprocessing
+import sys
+
+
+if getattr(sys, 'frozen', False):
+    multiprocessing.freeze_support()
+
+
+def create_window():
+    window = webview.create_window('Simple browser', 'https://pywebview.flowrl.com/hello')
+    webview.start()
+
+
+if __name__ == '__main__':
+    multiprocessing.set_start_method('spawn')
```

Comment:
remove this line.
on MacOS and Windows it will use 'spawn'
on Linux it will use `fork`(better method)
when frozen it will use `fork`